plugins/UserCountryMap/javascripts/realtime-map.js

```diff
@@ -141,7 +141,7 @@
                     filter_limit: maxVisits,
                     showColumns: ['latitude', 'longitude', 'actions', 'lastActionTimestamp',
                         'visitLocalTime', 'city', 'country', 'countryCode', 'referrerType', 'referrerName',
```

Comment:
Should we also escape referrer & city? I think those two can also be specified through tracker query params.

Reply:
doesn't hurt to do so, even if city can only be set when providing a token_auth

Reply:
changed it.